style.css

```diff
@@ -1,3 +1,230 @@
+/* GENERIC STYLING */ 
+
 * {
 	box-sizing: border-box;
+	font-family: Helvetica;
+	font-size: 10px;
+}
+
+h1 {
+	font-family: cursive;
+	color: #ebedf0; /*a grey-ish white*/
+	font-size: 7rem;
+	margin: 20px auto;
+}
+
+h2 {
+	font-size: 3rem ;
+	color: #d65a64; /* cute pink colour*/	
+	margin: 0 50px;
+}
+
+h3 {
+	color: #ebedf0;	
+	font-size: 2rem;	
+}
+
+p {
+	color: #ebedf0;
+	font-size: 1.8rem;
+}
+
+a {
+	text-decoration: none;
+	font-weight: bold;
+	color: #d65a64; 
+	font-size: 1.8rem;
+}
+
+.aboutMe_section, .portfolio_section, .contact_section {
```

Comment:
it is better to call the class `about_me_section` because it's not usually good to mix camelCase with snake_case or any case with any case for that matter